### ISSUE TITLE
Scrubbing on audio files doesn't seem to work

### ISSUE THREAD

**Issue by farski** [PRX/publish.prx.org#454]

**Comment by sandikbarr** [comment]
I am not having this problem in Chrome with the embeddable player or with the Audio Files upload on Episode -> Basic Info. Was this bug with a certain browser, audio files, something else?

**Comment by farski** [comment]
Seems to happen right after a file has been uploaded prior to the page being refreshed.

**Comment by pkarman** [comment]
Is the fix here as simple as automatically refreshing the page after an upload is complete?

**Comment by sandikbarr** [comment]
Yeah, I'm seeing that same behavior now uploading a new file. @pkarman that feels like cheating! 😈  We should probably look into it and see if we can see what's causing it before deciding how it can/should be fixed. Estimate of 3 seems to account for those unknowns. 👍
